features/logistics/composite-products.md

```diff
@@ -0,0 +1,21 @@
+# Composite Products
+
+## Overview
+
+There are industries (like furniture and fixtures) where businesses are accustomed to storing, handling, and distributing **products composed of individual parts or components** that may be stored and manipulated **separately** from one another. 
```

Comment:
"are accustomed" - unnecessary and not very adequate here. Just remove it. "businesses are storing, handling, ...."

Reply:
Removed.